src/goose/client.clj

```diff
@@ -166,7 +166,7 @@
   [opts cron-opts execute-fn-sym & args]
   (register-cron-schedule opts cron-opts execute-fn-sym args))
 
-(defn perform-batch
+(defn perform-batch-async
   ([opts batch-opts execute-fn-sym args-coll]
```

Comment:
question: I think this the braces around [opts batch-opts execute-fn-sym args-coll] is a mistake?